draft-ietf-tls-tls13.md

```diff
@@ -1673,7 +1668,7 @@ the protocol.
           } Random;
 
 random_bytes
-: 32 bytes generated by a secure random number generator.
+: 32 bytes generated by a secure random number generator. See D.1 for additional information.
 {:br }
```

Comment:
The glossary was removed, so now it's C.1. Link directly to the section instead to avoid the issue.